propka/bonds.py

```diff
@@ -28,11 +29,9 @@ def __init__(self):
         self.max_sq_distance = max(list(self.distances_squared.values())+[self.default_dist_squared])
 
         # protein bonding data
-        self.data_file_name = pkg_resources.resource_filename(__name__, 'protein_bonds.dat')
-
-        data = open(self.data_file_name,'rb')
-        self.protein_bonds = pickle.load(data)
-        data.close()
+        self.data_file_name = pkg_resources.resource_filename(__name__, 'protein_bonds.json')
```

Comment:
With this change, increase the minor release number in https://github.com/jensengroup/propka-3.1/blob/91925bb92092d3b180a9c086e81822a3fdec6939/setup.py#L9

Reply:
Addressed in 0fbaa6711ff0cd9777363108989cbe71d42dd60e.